src/resolvers/index.ts

```diff
@@ -39,13 +39,7 @@ const resolvers = {
   },
   Item: {
     savedItem: async (item: Item, args, context: IContext) => {
-      const save = await context.dataLoaders.savedItemsByUrl.load(
-        item.givenUrl
-      );
-      if (save == null) {
-        throw new NotFoundError(`No Save found for url=${item.givenUrl}`);
```

Comment:
we shouldnt be throwing a not found error here. we should just return null, and continue on. There are plent of cases where an Item won't return a savedItem and its not an error